rasa/server.py

```diff
@@ -779,14 +780,25 @@ def _get_events_from_request_body(request: Request) -> List[Event]:
             with app.active_training_processes.get_lock():
                 app.active_training_processes.value += 1
 
-            model_path = await train_async(
+            info = dict(
```

Comment:
Really wish `run_in_executor` also accepted `**kwargs`! Using `partial` was a good workaround 👍

Reply:
That's also the suggested way in the docs (Ella pointed it out to me :-) )